runtime/tools/python/ttrt/__init__.py

```diff
@@ -224,19 +44,25 @@ def main():
         choices=sorted(list(read_actions.keys())),
         help="output sections of the fb",
     )
-    read_parser.add_argument("binary", help="flatbuffer binary file")
+    read_parser.add_argument("--binary", default="", help="flatbuffer binary file")
     read_parser.set_defaults(func=read)
 
+    """
+    API: run
+    """
     run_parser = subparsers.add_parser("run", help="run a flatbuffer binary")
     run_parser.add_argument(
         "-p",
         "--program-index",
         default=0,
         help="the program inside the fbb to run",
     )
-    run_parser.add_argument("binary", help="flatbuffer binary file")
+    run_parser.add_argument("--binary", default="", help="flatbuffer binary file")
```

Comment:
Sorry I missed this one last time around.  I kind of liked having the binary be a positional argument since it kind of forces it to be required and is the subject of the verb (the command).  Lmk if there is a reason for doing it this way.

Reply:
hm yeah I agree. I ran into a small problem with the perf csv file but I can change it back.